### DISCUSSION TITLE
"Why are you not on campus?" keeps showing up

### DISCUSSION THREAD

**Discussion by pickaxe828** [community/community#146794] [GitHub Education]
### Hi everyone,

I am literally at school and using the computer from school, but I kept getting this question, and my application keeps getting rejected.
Meanwhile, "The school you selected does not appear to have a campus location in your country. " is also false, as I am at school right now, physically.

**Comment by davevad93** [comment]
Hi @pickaxe828 ,

It's possible that GitHub has wrong information about your school, try contacting support by opening a ticket [here](https://support.github.com/contact/education), select `My selected school has incorrect or incomplete information e.g. domains or campus location` and explain your issue.
